src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/SpecifyIFormatProvider.cs

```diff
@@ -174,65 +175,78 @@ protected override void InitializeWorker(CompilationStartAnalysisContext context
                 #endregion
 
                 #region "IFormatProviderAlternateStringRule & IFormatProviderAlternateRule"
-
-                IEnumerable<IMethodSymbol> methodsWithSameNameAsTargetMethod = targetMethod.ContainingType.GetMembers(targetMethod.Name).OfType<IMethodSymbol>().WhereMethodDoesNotContainAttribute(obsoleteAttributeType).ToList();
-                if (methodsWithSameNameAsTargetMethod.HasMoreThan(1))
                 {
```

Comment:
It looks like this set of braces can be removed

Reply:
Interestingly, it looks like Github moved your comments to different line numbers.